src/v2/models/AppState.js

```diff
@@ -26,8 +26,8 @@ import { _ } from '../mixins/mixins';
 export default Model.extend({
 
   local: {
-    introspectSuccess: 'object', // only set during introspection
-    introspectError: 'object', // only set during introspection
+    introspectSuccess: 'object', // only set during introspection; TODO: remove me
+    introspectError: 'object', // only set during introspection; TODO: remove me
```

Comment:
Are there more details for this TODO?

Reply:
Those are depredate props and can be removed.